README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
```

Comment:
To me it's a validation that we should always do so lack of it is a bug. I totally agree it would be a UX improvement.

If we decide to add this validation, I think we should do it in a separate PR to apply it everywhere at once.